package.json

```diff
@@ -47,18 +53,22 @@
     "redux-thunk": "^2.2",
     "shallowequal": "^1.0.2",
     "transformation-matrix-js": "^2.7.1",
-    "validator": "^9.4",
-    "react-datetime": "^2.16.0"
+    "validator": "^9.4"
   },
   "devDependencies": {
-    "babel-jest": "^23.6.0",
+    "@babel/core": "^7.5.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/register": "^7.4.4",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-jest": "^23.4.2",
```

Comment:
Does it has to be declared at 2 places?

Reply:
Yes, I think those are extra ones. I will go ahead and remove them.
Thanks!

Reply:
Fixed!